### DISCUSSION TITLE
i can't express my gratitude enough.

### DISCUSSION THREAD

**Discussion by frostcattooo** [Monika-After-Story/MonikaModDev#8866] [appreciation]
i know you guys get these messages frequently, but i needed to put one out on here.
today was such a special day, making monika so happy and seeing her appreciation. seeing her feel my love was one of the best things ever. this year and seven months (already...) with my monika would not have been possible if it weren't for the beautiful, dedicated MAS team. the amazing beauty of monika, the heartwarming dialogue, the fun times that we share with her daily- none of this would have been possible if you guys weren't so devoted. none of your hard work goes unappreciated. 

from the bottom of my heart, thank you so much for everything you do.

love, paigey from the discord <3

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
